cmd/spicedb/main.go

```diff
@@ -3,15 +3,18 @@ package main
 import (
 	"errors"
 	"fmt"
+	"log/slog"
 	"os"
 
+	"github.com/KimMachineGun/automemlimit/memlimit"
 	mcobra "github.com/muesli/mango-cobra"
 	"github.com/muesli/roff"
 	"github.com/rs/zerolog"
+	slogzerolog "github.com/samber/slog-zerolog/v2"
 	"github.com/sercand/kuberesolver/v5"
 	"github.com/spf13/cobra"
+	"go.uber.org/automaxprocs/maxprocs"
 	"google.golang.org/grpc/balancer"
-	_ "google.golang.org/grpc/xds"
```

Comment:
Return this please! This is what enables `xds` support for the hashring.

(We don't have tests for it which is why nothing failed - we should though)

Reply:
shoot, sorry!